### ISSUE TITLE
Changes in .menta don't trigger site rebuild while using nikola auto

### ISSUE THREAD

**Issue by jjconti** [getnikola/nikola#1967]
Steps:

0. nikola auto -b
1. nikola import_feed http://feeds.feedburner.com/EnElExilio
(site is regenerated using default author for posts)
2. Edit a post .meta file adding author

Nothing happens. If I delete output/* and nikola build, the posts are generated using the updated author in the posts with .meta updated.

**Comment by jjconti** [comment]
Also, If I change the author in one of these posts, I need to delete the output and regenerate to have the author updated in the output post.

**Comment by ralsina** [comment]
1. Please import the site before starting nikola auto, it makes no sense to do it the other way around, because you are changing the config that decides what auto watches, and it's getting imported into new_site so it's just hard to test.

2. Do you have *.html  in your POSTS setting in conf.py? When I have that it works perfectly here:

```
(nikola)ralsina@yoga:/tmp/new_site$ vim posts/20071127jargon-fortunes.meta
(nikola)ralsina@yoga:/tmp/new_site$ [2015-08-24T01:32:57Z] INFO: auto: REBUILDING SITE (from posts/20071127jargon-fortunes.meta)
Scanning posts.....done!
.  render_posts:timeline_changes
.  render_posts:cache/posts/20071127jargon-fortunes.html
.  render_archive:output/2007/index.html

[2015-08-24T01:32:58Z] INFO: auto: REFRESHING: output/2007/index.html
[2015-08-24T01:32:58Z] INFO: auto: ---> /2007/index.html
[2015-08-24T01:32:58Z] INFO: ws4py: Terminating websocket [Local => 127.0.0.1:8000 | Remote => 127.0.0.1:41530]
[2015-08-24T01:32:58Z] INFO: ws4py: Managing websocket [Local => 127.0.0.1:8000 | Remote => 127.0.0.1:41552]
[2015-08-24T01:32:58Z] INFO: auto: <--- {u'snipver': 1, u'ver': u'2.2.1', u'command': u'hello', u'protocols': [u'http://livereload.com/protocols/official-6', u'http://livereload.com/protocols/official-7']}
[2015-08-24T01:32:58Z] INFO: auto: ---> {"serverName": "nikola-livereload", "command": "hello", "protocols": ["http://livereload.com/protocols/official-7"]}
[2015-08-24T01:32:58Z] INFO: auto: <--- {u'url': u'http://127.0.0.1:8000/', u'command': u'info', u'plugins': {u'less': {u'disable': False, u'version': u'1.0'}}}
[2015-08-24T01:32:58Z] INFO: auto: ****** Browser connected: http://127.0.0.1:8000/
[2015-08-24T01:32:58Z] INFO: auto: ****** sending 0 pending messages
```

**Comment by ralsina** [comment]
OTOH, the author is not getting refreshed in the output, so there is a real bug in there.

**Comment by ralsina** [comment]
The problem is a lot more general that this. Since I already filed some data about it in another issue, I am closing this one as duplicate of https://github.com/getnikola/nikola/issues/1968
